bindings/python/tests/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ endif()
 
 set(test_python_bindings_DATA
         runTests.py.in
+	test_gettext.py
```

Comment:
Cosmetic: can you use spaces instead of a tab here ? That will align the file with the others.

Reply:
Sure. I'm using vim and on python files that gets done automatically. But this is a txt.